### PR TITLE
graph-veracode -> graph-veracode-v2

### PR DESCRIPTION
**Pull request by mdaum**

### CODE REVIEW

**Comment by mdaum** [comment]
https://github.com/JupiterOne/graph-veracode
https://github.com/JupiterOne/graph-veracode-v2
we deprecated `graph-veracode` some time ago. however the docs were being overwritten by the deprecated graph project every 24 hours. This should resolve the issue.

File: knowledgeBase/APIs_and-integrations/application-security/graph-veracode-v2.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+### Relationships
+
+The following relationships are created:
+
+| Source Entity `_type`  | Relationship `_class` | Target Entity `_type`  |
+| ---------------------- | --------------------- | ---------------------- |
+| `veracode_account`     | **HAS**               | `veracode_application` |
+| `veracode_application` | **IDENTIFIED**        | `veracode_finding`     |
+
+### Mapped Relationships
```
Comment (mdaum) [inline knowledgeBase/APIs_and-integrations/application-security/graph-veracode-v2.md:132]: do the mapping docs get dynamically updated? The file was moved not too long ago so I cannot tell. The mapping docs for [veracode](https://github.com/JupiterOne/docs/blob/vanilla-staging/knowledgeBase/jupiterOne-data-model/mappings-integrations.md#veracode-mappings) are no longer accurate. Happy to rewrite if this is done manually.

**Review by janettelynch — APPROVED** [review]

**Comment by mdaum** [comment]
We are actually going to change `graph-veracode-v2` to `graph-veracode`, and rename the old `graph-veracode` to `graph-veracode-legacy`. We'll let the nightly sync take care of things from here.

**Comment by janettelynch** [comment]
@mdaum Is it possible to carry over the edits I made this morning to the new graph-veracode?

**Comment by mdaum** [comment]
@janettelynch it is possible, but a PR should be made to graph-veracode modifying [this file](https://github.com/JupiterOne/graph-veracode/blob/main/docs/jupiterone.md)

**Pull request closed without merging**